src/components/Contact.js

```diff
@@ -2,13 +2,22 @@ import React, { Component } from 'react';
 import { Jumbotron } from 'react-bootstrap';
 
 export default class Contact extends Component {
-    render() {
+  render() {
         return (
             <div className="row justify-content-center">
                 <Jumbotron className="Jumbotron-Global-Settings">
                     <div className="row justify-content-center">
-                        <h2 className="Contact-Header col-12">Get in touch to start planning the perfect cake for your occasion!</h2>
+                      <h2 className="Contact-Header col-12">Get in touch to start planning the perfect cake for your occasion!</h2>
                     </div>
+                      <div>
+                        <from>
+                          <label htmlFor="name" className="Contact-Name">Name:</label>
+                          <input name="Name" type="text" required placeholder="Your Name" className="Contact-Name-Input"/>
+                          <label htmlFor="email" className="Contact-Email">Email:</label>
+                          <input name="Email" type="text" required placeholder="Email" className="Contact-Email-Input"/>
+                          <button type="submit" className="Contact-Submit-Button">Get In Touch</button>
+                        </from>
```

Comment:
While the basic form is helpful, I'd like it to meld a little better with the UI theme. I'll txt you some examples of how sites manage styles for their forms.

Reply:
Please make form items inline with their labels; this helps the visuals resolve a little easier. Also, the rounding on the fields makes the placeholder text look a bit cramped. Perhaps the rounding could be reduced or the margin for the text could be increased?